Source/Lib/Codec/EbDefinitions.h

```diff
@@ -12,7 +12,6 @@ extern "C" {
 #endif
 
 #define PAREF_OUT 1 // Disconnect pa ref  from input for both single/multi core
-#define OUT_ALLOC 0 // Output bitsream allocation at run time for both single/multi core
 
```

Comment:
It's better to temporarily keep the macro here, for some issue's regression checking more easily, just in case.

Reply:
with output buffer size increased to 5000, we can use the old logic anymore. AV1 also removed out_alloc macro.